playbooks/archive_artifacts.yml

```diff
@@ -0,0 +1,77 @@
+- hosts: "{{ hosts }}"
+  tasks:
+    - name: Create temporary directory to store artifacts
+      file:
+        path: "{{ artifacts_path }}"
```

Comment:
I thought rsync couldn't create the {{ artifacts_path }}/{{ ansible_hostname }}, just other child directories of "the base dir". I guess I am wrong.